exercises/practice/dnd-character/.meta/example.py

```diff
@@ -14,8 +14,7 @@ def __init__(self):
 
     def ability(self):
         dice_rolls = sorted(
-            [random.randint(1, 6)
-                for _ in range(4)])
+            [random.randint(1, 6) for _ in range(4)])
```

Comment:
Since this is on two lines, I can't do a suggestion, but since `sorted()` unpacks and it also returns a list, it is not necessary to put a list in as an argument -- it can be a generator expression.

Reply:
Will do! 💙